docs/administration.md

```diff
@@ -349,10 +349,24 @@ When you use the provided docker compose script, put the export inside
 the `export` folder in your paperless source directory. Specify
 `../export` as the `source`.
 
+If you have zipped the data and manifest 
+with the [Document exporter](#exporter) using the `-z` or `--zip` option, please
+extract the files inside the `export` folder first.
+
```

Comment:
```suggestion
Note that .zip files (as can be generated from the exporter) are not supported.

```

Everything else here is extraneous, repeating the cli flags, mentioning the data and manifest, etc.